src/elements/emby-checkbox/emby-checkbox.css

```diff
@@ -46,14 +47,13 @@
     position: absolute;
     top: 3px;
     left: 0;
-    display: inline-block;
```

Comment:
uh? why remove?

Reply:
It has a display: flex further below. Can't have more than one display type.